index.js

```diff
@@ -57,6 +80,49 @@ function assembleStyles() {
 		});
 	});
 
+	function rgb2rgb(r, g, b) {
+		return [r, g, b];
+	}
+
+	styles.color.close = '\u001b[39m';
+	styles.bgColor.close = '\u001b[49m';
+
+	styles.color.ansi = {};
+	styles.color.ansi256 = {};
+	styles.color.ansi16m = {
+		rgb: wrapAnsi16m(rgb2rgb, 0)
+	};
+
+	styles.bgColor.ansi = {};
+	styles.bgColor.ansi256 = {};
+	styles.bgColor.ansi16m = {
+		rgb: wrapAnsi16m(rgb2rgb, 10)
+	};
+
+	for (var key in colorConvert) {
+		if (!colorConvert.hasOwnProperty(key)) {
+			continue;
+		}
+
+		var fn = colorConvert[key];
+		var conversion = key.replace(/^([^2]+).+$/, '$1');
```

Comment:
You can in theory, but the API may not support it.

Further, `for (var k in convert['ansi256'])` would return a bunch of keys to a bunch of functions _from_ 256 colors, which doesn't really help us. We'd have to cache them and then reverse-look them up too, which is much messier than two loops.